src/governance.cpp

```diff
@@ -1029,17 +1026,76 @@ void CGovernanceManager::CheckMasternodeOrphanObjects()
             continue;
         }
 
-        bool fAddToSeen = true;
-        if(AddGovernanceObject(govobj, fAddToSeen)) {
-            LogPrintf("CGovernanceManager::CheckMasternodeOrphanObjects -- %s new\n", govobj.GetHash().ToString());
-            govobj.Relay();
-            mapMasternodeOrphanObjects.erase(it++);
-        }
-        else {
+        AddGovernanceObject(govobj);
+        mapMasternodeOrphanObjects.erase(it++);
+    }
+}
+
+void CGovernanceManager::CheckPostponedObjects()
+{
+    LOCK2(cs_main, cs);
+
+    // Check postponed proposals
+    for(object_m_it it = mapPostponedObjects.begin(); it != mapPostponedObjects.end();) {
+
+        const uint256& nHash = it->first;
+        CGovernanceObject& govobj = it->second;
+
+        assert(govobj.GetObjectType() != GOVERNANCE_OBJECT_WATCHDOG &&
```

Comment:
I noticed [here](https://github.com/the-schnibble/dash/blob/f505afe1615a8031bfd8a1a4cb2364849a5813e3/src/governance-object.cpp#L468) that triggers and watchdogs are considered as exceptions and all other objects must meet collateral confirmations. Just now there are only proposals...